about.html

```diff
@@ -0,0 +1,154 @@
+            </li>
+          </ul>
+          <span class="campaign">
+            <a href="#"> CC Campaign </a>
+          </span>
+        </nav>
+      </section>
+
+      <section class="banner">
+        <h1>ARTWORK since many years</h1>
+                <p class="banner-description">This art exhibition has played a crucial part in the market for new art since the 18th and 19th centuries.
+                        The Paris Salon, open to the public from 1737, rapidly became the key factor in determining the reputation, 
+                        and so the price, of the French artists of the day. </p>
+       
+      </section>
+    </header>
+
+   
+    <div class="container section-product">
+      <h3 class="d-flex justify-content-center card-det">Artworks events - 2022
+         </h3>
+      <div class="container d-flex my-3 justify-content-center line-height"></div>
+      <p class="container text-center d-flex justify-content-center text-card">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque sapiente placeat nam corrupti dolores
+          culpa excepturi ducimus quasi magni illum. Quibusdam facere consequatur odio sit sint dolores numquam
+          iusto at?
+      </p>
+      <div class="logo-about d-flex justify-content-center"><img src="./img/artevent.png" alt="logo"></div>
+  </div>
+  <div class="pt-5 rank-section-about">
+      <h3 class="d-flex justify-content-center title-rank">We started in 2013</h3>
+      <div class="container d-flex my-3 justify-content-center line-height"></div>
+      <p class="container d-flex justify-content-center text-center rank-desc">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Itaque sapiente placeat nam corrupti dolores
+          culpa excepturi ducimus quasi magni illum. Quibusdam facere consequatur odio sit sint dolores numquam
+          iusto at?
+      </p>
+
+      <div class="production-container container-fluid testymonies mb-5 testymonies2">
+      </div>
+  </div>
+  <main>
+      <section class="partners">
+            <h2>Partners</h2>
+            <div class="partners-underline"></div>
+            <div class="partners-logo">
+                <img class="ptners-logo" src="./img/logogray.png" alt="">
+                <img class=" ptners-logo" src="./img/logogray4.png" alt="">
+                <img class="ptners-logo" src="./img/logogray.png" alt="">
+                <img class="ptners-logo" src="./img/logogray4.png" alt="">
+            </div>
+          </section>
+          <footer>
+            <div class="footer-container">
+                <div class="col-md-3 py-5 text-center footer-title">© ART EXHIBITION</div>
+                <div class="col-md-9 p-5">
+                    <h5 class="pb-3 mt-3 footer-params">2022 - Art exhibitions represent the gathering of art objects into a space for a temporary event. Some rights reserved </h5>
+      
+                  </div>
+            </div>
+          </footer>
```

Comment:
- [ ] [OPTIONAL] kindly make the background of your about footer to be black on desktop version so that you match the Behance design.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181689549-8198291a-b41c-4327-a3a0-57acb7d2026f.png)

**This is how it is supposed to look like:**

![image](https://user-images.githubusercontent.com/86318284/181689629-e107f0fe-a343-43c3-9f3f-d01b4355ef28.png)

- [ ] Also kindly note that in the about page desktop version, the "partners section" is not supposed to be shown.

**This is the Behance design**

![image](https://user-images.githubusercontent.com/86318284/181689817-8b8d04f6-443e-4095-a723-4be3f6f9328a.png)


**This is your design:**

![image](https://user-images.githubusercontent.com/86318284/181689869-a000c215-0464-419f-ac6f-7626de8f6307.png)